docs/dictionary/command/go.lcdoc

```diff
@@ -36,6 +36,13 @@ go back 7
 Example:
 go invisible stack "Preferences"
 
+Example:
+put the filename of me into tStackFile
+set the itemDelimiter to "/"
+put item 1 to -2 of tStackFile into tStackFolder
+set the defaultFolder to tStackFolder
+go stack somethingElse.livecode
```

Comment:
Could you make sure this compiles in strict mode please? I.e. declare the variables and put `somethingElse.livecode` in quotes? Also I think a line of comment like 
```
-- somethingElse.livecode is a stackfile in the same folder as this stack
```
would help explain what the example is for.